src/components/Sidebar/Sidebar.tsx

```diff
@@ -46,15 +46,23 @@ const SidebarNav = styled.nav`
     }
   }
 `;
+const VercelStyle = styled.a`
+  height: 0px;
+  width: 40px;
+  marginright: 10px;
+  color:#f90606 @media ${desktop} {
+    display: none;
+  }
+`;
```

Comment:
This section has a few bugs in it.  

- I assume `marginright` should be `margin-right`.
- I don't think `color:#f90606 @media ${desktop} {` is doing what you intended, but also, if it did, it would mean that the vercel logo would not display on `desktop`.  We'd like it to show on all browsers.

Reply:
moved to the bottom and working on the bugs